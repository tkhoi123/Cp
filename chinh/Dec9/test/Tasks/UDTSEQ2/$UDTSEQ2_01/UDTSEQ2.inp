6 4
2 6 -29551
3 4 -12941
1 4 21115
2 4 -14415
