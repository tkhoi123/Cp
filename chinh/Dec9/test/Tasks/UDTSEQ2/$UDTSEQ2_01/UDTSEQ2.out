21115 999998264 999962472 999926680 999881796 999852245 